server/etcdserver/server.go

```diff
@@ -1223,6 +1223,15 @@ func (s *EtcdServer) TransferLeadership() error {
 		return nil
 	}
 
+	if active, safe := s.isQuorumActiveAfterOneMemberIsDown(0 /* healthInterval */); !safe {
```

Comment:
Why zero? I think requiring member to be available for at least `HealthInterval` is better heuristic for whether cluster is able to progress. Let's leave it the same as `mayRemoveMember` function. Might just remove this as function argument/

Reply:
This check is to avoid unnecessary leadership transfer. With `0` HealthInterval, this still has higher chance of transfer leadership to reduce the disruption of no leader downtime. 

So I think `0` is better. WDTY?